wrappers/rust/oneds-telemetry/src/lib.rs

```diff
@@ -0,0 +1,161 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+use log::debug;
+use once_cell::sync::Lazy;
+use std::{sync::RwLock, ffi::CString};
+use types::{StringProperty, TelemetryItem, TelemetryProperty};
+use cpp_client_telemetry_sys::{
+    evt_open,
+    evt_close,
+    evt_log,
+    evt_flush,
+    evt_upload,
+    evt_prop
+};
+
+pub mod appender;
+pub mod types;
+
+#[derive(Clone)]
```

Comment:
Is `log_handle` a pointer or a handle that was received from the underlying C API?  If so, then I don't think this Clone call is correct or safe.